tools/virtual_scanner.cpp

```diff
@@ -174,7 +174,6 @@ main (int argc, char** argv)
 
   int subdiv_level = 1;
   double scan_dist = 3;
-  std::string fname, base;
```

Comment:
fname is used somewhere else :)

Reply:
Thanks @kunaltyagi for pointing out. Will fix it right away.